examples/create_synthesizer.py

```diff
@@ -9,7 +9,7 @@
 
 
 # Create a query function for a target PyTorch Lightning model
-model = train_four_layer_mnist_victim(gpus=1)
+model = train_four_layer_mnist_victim(gpus=torch.cuda.device_count())
```

Comment:
Add a comment explaining how this is different from manually setting the gpus argument to 1 or 0.